src/util/sqlhelper.cpp

```diff
@@ -15,7 +16,7 @@ std::ostream& operator<<(std::ostream& os, const DatetimeField& datetime);
 std::ostream& operator<<(std::ostream& os, const FrameBound& frame_bound);
 
 std::string indent(uintmax_t num_indent) { return std::string(num_indent, '\t'); }
-void inprint(int64_t val, uintmax_t num_indent) { std::cout << indent(num_indent).c_str() << val << "  " << std::endl; }
+void inprint(std::int64_t val, uintmax_t num_indent) { std::cout << indent(num_indent).c_str() << val << "  " << std::endl; }
```

Comment:
This is an example why I don't like the prefix. We would also need to add it to many other things (here uintmax_t) to make it consistent, right?